### DISCUSSION TITLE
Documentation about how to update after using the install script

### DISCUSSION THREAD

**Discussion by Zoobdude** [immich-app/immich#8047] [General]
Currently there is no documentation about how a user should update after using the [install script](https://immich.app/docs/install/script) in the docs or in the actual script. Seeing as those who are using the install script are probably less familiar with Docker/self-hosting, I think specific instructions may be beneficial.

**Comment by nicomerl** [comment]
While i do understand your worries the install script does not "install" a software how a executable would do. It just downloads and starts the containers. With selfhosting you should at least have a basic understanding of whats happenig because one day there will be smaller or bigger troubles. 

But to also help you: 
- with docker-compose there is 2 files that are relevant: docker-compose.yml and .env
- in the .env-File in line 7 you can see the Immich Version. By Default this is "release". Release ist always the latest deployed container with the "release-tag". 
- See https://hub.docker.com/r/altran1502/immich-server/tags. You can see that right now v1.99.0 and release share the same "digest" or Version Hash. So with the latest deployment you are on this version.  

Updateprocedure as following:
- change directory to your immich docker Folder
- docker compose stop (stops all running immich containers)
- docker compose pull (pulls newer version when "release"-Version has been updated)
- docker compose up -d (starts all immich containers)
